### DISCUSSION TITLE
Icon to fast access

### DISCUSSION THREAD

**Discussion by egolfeld** [isontheline/pro.webssh.net#482] [Ideas]
Prompt 2 has interesting feature, when it launched it’s shows icon, so users can switch to other apps and return back very fast
<img width="340" alt="Изображение PNG" src="https://user-images.githubusercontent.com/35452983/146833234-db3e3e58-ccf5-4383-a9b1-a1447ecf4968.png">

**Comment by isontheline** [comment]
I don't understand this feature, could you explain more about it ?

Thanks 🙏

**Comment by egolfeld** [comment]
Pretty easy idea, just an icon to open webssh, hovever, maybe no one really needs it :)


https://user-images.githubusercontent.com/35452983/146904081-7cefaf01-8ea8-4e71-9a30-57d326979b06.MP4

**Reply by isontheline** [reply]
@egolfeld But the icon you are pressing is because Prompt is using GPS in order to fake Apple system to don't sleep the app and don't terminate the connection. 

Isn't it?

If so I won't implement a such feature because I love batteries and I love privacy by design ❤️

**Reply by egolfeld** [reply]
Hm, interesting, didn't realized it as GPS, if so, it's very bad

Another reason to move from prompt 2 :)

**Reply by egolfeld** [reply]
Thank you for clarification

**Reply by isontheline** [reply]
Your welcome 😄 

I don't have bought Prompt 2 so I can't make any judgment :innocent:

I think they have used the "best" ability (hack) offered (tolerated) by the iOS in order to satisfy users who want to keep their connections open more than the 30 seconds allowed by Apple.

Maybe the time will come we will be allowed to run connections more than 30 seconds natively... 🙏 I hope
